modules/lambdas/main.tf

```diff
@@ -146,6 +147,55 @@ resource "aws_lambda_function" "extract_filepaths_for_granule" {
 }
 
 
+data "aws_iam_policy_document" "assume_lambda_role_extract" {
+  statement {
+    principals {
+      type = "AWS"
+      identifiers = ["*"]
```

Comment:
The identifier should be `["lambda.amazonaws.com"]`

See similar example here https://github.com/nasa/cumulus-orca/blob/4d6cff91690984c4a8b947d89a2115d1e0009301/modules/iam/main.tf#L17

Reply:
Fixed this, the AWS tech said I needed to have both "lambda.amazonaws.com" and "states.amazonaws.com" set here

Reply:
sounds good